blogs/comm-opt/README.md

```diff
@@ -0,0 +1,94 @@
+<div align="center">
+
+# Communictaion Optimization for Large-Scale Training 
+
+</div>
+
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [Parameter-update All-Reduce Optimization](#ar-opt)
+3. [ZeRO2 Gradient All-Gather Optimization](#ag-opt)
+4. [Sequence-Parallel Optimization](#sp-opt)
+5. [Sequence-Parallel Convergence Consideration](#convergence)
+
+
+## 1. Introduction <a name="introduction"></a>
+Training LLMs on large data-set can be costly both in terms of resource-budget and time-to-completion. An important step to minimize such cost is to find the right number of resources together with a scalable library that assures training under a time limit. In this post, we target a key component of the scalability feature at DeepSpeed, the communication optimization. There are several communication collectives used in the main DeepSpeed's technologies, which we go through their optimization in detail in the following sections.
+
+## 2. Parameter-update All-Reduce Optimization <a name="ar-opt"></a>
```

Comment:
I believe this should be gradient all-reduce, not parameter-update all-reduce.

Reply:
I think a better title would be something like "Optimizing Gradient Reduction for Mixture of Expert training".

Reply:
I think this relates to both dense and MoE architecture. Of course the second approach is only for MoE models, but the first one can apply for all the architectures.